ios/BrightcovePlayer.m

```diff
@@ -129,20 +143,110 @@ - (void)playbackController:(id<BCOVPlaybackController>)controller playbackSessio
         if (self.onReady) {
             self.onReady(@{});
         }
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("playbackReady")
+                                 });
+        }
     } else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventPlay) {
         _playing = true;
         if (self.onPlay) {
             self.onPlay(@{});
         }
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("playbackPlay")
+                                 });
+        }
     } else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventPause) {
         _playing = false;
         if (self.onPause) {
             self.onPause(@{});
         }
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("playbackPause")
+                                 });
+        }
     } else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventEnd) {
         if (self.onEnd) {
             self.onEnd(@{});
         }
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("playbackEnd")
+                                 });
+        }
+    } else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventFail) {
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("loadFail"),
+                                 @"error": lifecycleEvent.properties[kBCOVPlaybackSessionLifecycleEventError]
+                                 });
+        }
+    } else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventFailedToPlayToEndTime) {
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("failedToPlayToEndTime"),
+                                 @"error": lifecycleEvent.properties[kBCOVPlaybackSessionLifecycleEventError]
+                                 });
+        }
+    }  else if (lifecycleEvent.eventType == kBCOVPlaybackSessionLifecycleEventResumeBegin) {
+        if (self.onStatusEvent) {
+            self.onStatusEvent(@{
+                                 @"type": @("resumeAttemptBegin")
```

Comment:
Can you remove the "attempt" from these so they match the event type better. Similar to how the other strings are just the ending part. This one would be "resumeBegin".

Reply:
I did this on purpose to match the description and make it a little clearer for logging and when we go back to read the logs so we don't have to keep looking up what the words actually mean.  I'll change it for consistency.